src/pages/Cart/Cart.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import './Cart.scss';
+
+class Cart extends React.Component {
+  render() {
+    return (
+      <div className="Cart">
+        <div className="cartView">
+          <div className="cartTop">
+            <h1>장바구니</h1>
+            <div className="cartTopBtn">
+              <button type="button">+ 제품추가</button>
+              <button type="button">휴지통</button>
+            </div>
+          </div>
+
+          <h2>정기구독 제품</h2>
+          <div className="cartList">
+            <input type="checkbox" />
+            <img alt="이미지" src="http://localhost:3000/images/pill1.jpeg" />
```

Comment:
```suggestion
            <img alt="이미지" src="/images/pill1.jpeg" />
```
localhost:3000까지는 안적어줘도 됩니다. 안적어주면 현재 실행중인 서버주소로 자동으로 인식됩니다.
오히려 3000포트를 임의적으로 적어주게 되면 추후 3001포트, 3002포트등에서 실행됬을 경우 이미지가 제대로 안 불러와질 수 있습니다.

Reply:
alt에 이미지라고 적는것은 좋지 않습니다!
이미 img tag이기 때문에 이미지인것은 알 수 있습니다. 무슨 이미지인지 표현해주세요! e.g.) alt="약"

Reply:
수정 완료!!😎